Configuration/Generator/python/QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CP5Settings_cfi import *
```

Comment:
Hello, we noticed that this import has a wrong path and failed in production. It should be fixed soon. 
`from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *`

Reply:
see  https://github.com/cms-sw/cmssw/pull/33700